res/css/views/rooms/_MessageComposer.pcss

```diff
@@ -314,74 +314,6 @@ limitations under the License.
     }
 }
 
-.mx_MessageComposer_formatting {
```

Comment:
While updating the colors of the message action bar and composer formatting bar, I found that none of these classes are used anymore